addon/types/model_export/model.py

```diff
@@ -172,13 +177,36 @@ def generate_qc(self):
             qc.write('$mostlyopaque')
             qc.write('\n')
 
+        output_x = 0
+        output_y = 0
+        output_z = 0
+        output_rotation = 0
+        output_scale = 1
+
+        if self.custom_transform_source == 'MANUAL':
+            output_x = self.origin_x
+            output_y = self.origin_y
+            output_z = self.origin_z
+            output_rotation = self.rotation
+            output_scale = self.scale
+        elif self.custom_transform_source == 'CUSTOM_OBJECT' and self.custom_transform_object_ref is not None:
+            output_x = (self.custom_transform_object_ref.matrix_world[0][3]) * -1
+            output_y = self.custom_transform_object_ref.matrix_world[1][3]
+            output_z = (self.custom_transform_object_ref.matrix_world[2][3]) * -1
+            output_rotation = degrees(self.custom_transform_object_ref.rotation_euler.z)
+            # Only using X axis for scale, as the object is assumed to be uniformly scaled
```

Comment:
I don't usually use comments, but when I do, I end them with a period.
I think I agree with using X axis scale though yeah, simpler and more predictable than for example taking an average of scale across the axes.

Reply:
I misread this message, and so the comment is gone instead of just adding a full stop at the end, whoops.
If there's any need for further changes, I can sneak it back in there, perhaps.